samples/2_Cookbook/0_MatrixTranspose/Makefile

```diff
@@ -17,12 +17,11 @@ EXECUTABLE=./MatrixTranspose
 
 all: $(EXECUTABLE) test
 
-CXXFLAGS =-g
-CXX=$(HIPCC)
+CXXFLAGS =-O3
 
 
 $(EXECUTABLE): $(OBJECTS)
-	$(HIPCC) $(OBJECTS) -o $@
+	$(HIPCC) $(CXXFLAGS)  $(OBJECTS) -o $@
```

Comment:
Never noticed this before since the samples were building fine. But we don't seem to have a rule for building the OBJECTS. So either we add that or drop the OBJECTS completely from the makefile.